perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -348,9 +348,21 @@ sub by_category_ajax_data : Private {
         $body->{councils_text} = $c->render_fragment( 'report/new/councils_text.html');
     }
 
-    my $hints = $lookups->{hints}{$category};
-    $body->{title_hint} = $hints->{title} if $hints->{title};
-    $body->{detail_hint} = $hints->{detail} if $hints->{detail};
+    # only use cobrand title overrides if all bodies share a cobrand.
```

Comment:
A cobrand can't be on more than one body, see the check around line 500 in Admin/Bodies.pm, so if there's more than one body it can't be all the same cobrand, and I think these lines can be boiled down to `my $cobrand_overrides = $c->stash->{title_field_overrides_by_cobrand}->{$bodies->[0]->{name}} if @$bodies == 1;` (or something like that anyway).